packages/css/src/details.css

```
@@ -110,10 +105,34 @@
     interpolate-size: allow-keywords;
   }
 
+  &::details-content {
```

can do:
```suggestion
  &::details-content,
  &::part(details-content) {
    ...
  }
```
instead of repeating same twice? ☺️

---

Yeah i couldn't get that to work on Firefox.

---

hm, if it does not work, it might be because firefox does not understand one of the selectors. We can then use `&:is(::details-content, ::part(details-content))` instead, as `:is()` is forgiving if one of the selectors is unknown ☺️

---

Cool I’ll try that.

---

:is does not select pseudo elements 😞

---

Waaat. Æsj. Learning something new everyday 🤷‍♂️